add-on/chrome/bootstrap.js

```diff
@@ -211,6 +211,11 @@ var WindowListener = {
             });
           };
 
+          if (PrivateBrowsingUtils.isWindowPrivate(window)) {
+            resolve(null);
+            return;
+          }
```

Comment:
I'm also not sure if we should `reject()` instead as the UITour code will call an optional `aOpenCallback` on success: https://dxr.mozilla.org/mozilla-central/source/browser/components/uitour/UITour.jsm#1720-1742